specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/examples/IntegrationAccountLargeMaps_CreateOrUpdate.json

```
@@ -0,0 +1,66 @@
+{
+  "parameters": {
+    "api-version": "2019-05-01",
+    "resourceGroupName": "testResourceGroup",
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
```

Avoid using personal identifying information:

```suggestion
    "subscriptionId": "<Azure-subscription-ID>",
```